js/MVP/hooks/referat.js

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from 'react-query';
+import { getReferatPdf } from '../services/referat';
+
+const MOTEREFERAT_PDF = 'motereferatPDF';
+
+export const useMotereferatPDF = () => {
+  return useQuery(MOTEREFERAT_PDF, getReferatPdf, {
+    staleTime: 10000,
```

Comment:
Kan evnt fjerne denne, dersom dere legger på en høyere global default

Reply:
Enig, men denne hooken skal slettes til fordel for denne clunky koden: `export const downloadBrevPdf = async (uuid) => {
  const data = await getBrevPdf(uuid);
  const blob = await data.blob();

  const link = document.createElement('a');
  if (!link.download) {
    const url = URL.createObjectURL(blob);
    link.setAttribute('href', url);
    link.setAttribute('download', 'referat.pdf');
    link.style.visibility = 'hidden';
    document.body.appendChild(link);
    link.click();
    document.body.removeChild(link);
  }
};`

Som sikkert kan forbedres.